test/apps/test_fuzzy_match.py

```diff
@@ -40,12 +40,23 @@ def __init__(self, ucx_info, verbose):
         self.ucx_info = ucx_info
         if verbose:
             logging.basicConfig(level=logging.DEBUG)
+       
+    def ib_available(self):
+        status, output = commands.getstatusoutput('ibv_devinfo')
+        if status != 0:
+            return False
         
+        return 'No IB devices found' not in output
+         
     def run(self, expected):
-        with Environment(expected.keys()):
+        if expected['ib_required'] and not self.ib_available():
+            # Skip test if IB transport is required but not available.
```

Comment:
Excessive commenting

Reply:
you mean it's not needed at all? or it should just be shorter?